src/compiler/code_generator.ts

```diff
@@ -765,7 +788,11 @@ export class CodeGenerator {
 
       // note: this part is duplicated from end of compilemulti:
       const args = block!.children.map((c) => c.varName).join(", ");
-      this.insertBlock(`multi([${args}])`, block!, ctx)!;
+      if (ast.deepRemove) {
```

Comment:
same here, create a `args` variable and use it once instead of making a if statement